src/main/resources/migration/changelog-v5.5.0.xml

```diff
@@ -110,4 +110,27 @@
             <column name="OPERATION_MODE" type="VARCHAR(255)"/>
         </addColumn>
     </changeSet>
+
+    <changeSet id="v5.5.0-10" author="sahibamittal">
+        <createTable tableName="COMPONENT_PROPERTY">
+            <column autoIncrement="true" name="ID" type="BIGINT">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="COMPONENT_PROPERTY_PK"/>
+            </column>
+            <column name="COMPONENT_ID" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="GROUPNAME" type="TEXT"/>
+            <column name="PROPERTYNAME" type="TEXT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="PROPERTYVALUE" type="TEXT"/>
+            <column name="PROPERTYTYPE" type="TEXT">
+                <constraints nullable="false"/>
+            </column>
```

Comment:
Do we want a check constraint here, similar to what we have for `COMPONENT_CLASSIFIER_check` etc.?